src/workflow/ArcanistDiffBasedWorkflow.php

```diff
@@ -72,6 +72,16 @@ protected function validateStagingSetup() {
             }
         }
 
+        // Using staging area remotes retrieved from Phabricator as-is will
```

Comment:
Does uber.diff.staging.uri.override contains URI with `oc://`?
if so we can apply logic from https://github.com/uber/arcanist/pull/290 ?

Reply:
But getRepositoryQuery from #290 will always return replaced $remote_uri. Should we use `uber.diff.staging.uri.override` there?

Reply:
Make sense, thanks for clarification